packages/sitecore-jss-vue/src/components/RichText.test.ts

```diff
@@ -111,4 +112,85 @@ describe('<RichText />', () => {
     expect(rendered.attributes()).toMatchObject(attrs);
     expect(rendered.element.innerHTML).toBe(props.field.value);
   });
+
+  it('should initialize links', () => {
+    const props = {
+      field: {
+        value: '<div id="test"><h1>Hello!</h1><a href="/t10">1</a><a href="/t10">2</a></div>',
+      },
+    };
+    const rendered = mount(RichText, { props });
+    expect(rendered.element.innerHTML).toContain('<div id="test">');
+    expect(rendered.element.innerHTML).toContain('<h1>Hello!</h1>');
+    expect(rendered.element.innerHTML).toContain('<a href="/t10">1</a>');
+    expect(rendered.element.innerHTML).toContain('<a href="/t10">2</a>');
+  });
+
+  it('should not initialize links when does not have value', () => {
```

Comment:
This test not related to the links, it's just a test which check that nothing is rendered
you just need to rename it properly